native/cocos/renderer/pipeline/Define.h

```diff
@@ -158,12 +159,16 @@ enum class CC_DLL RenderQueueSortMode {
 };
 CC_ENUM_CONVERSION_OPERATOR(RenderQueueSortMode)
 
-struct CC_DLL RenderQueueDesc {
+class CC_DLL RenderQueueDesc : public cc::RefCounted {
```

Comment:
It has already been in `cc` namespace, `cc::` is redundant.

Reply:
yes

Reply:
fixed